src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -182,7 +181,7 @@ public final void testPathsDirectoryWithNestedFile() throws IOException {
         antTask.setProject(new Project());
 
         final FileResource fileResource = new FileResource(
-            antTask.getProject(), getPath(FLAWLESS_INPUT_DIR));
+            antTask.getProject(), getPath(FLAWLESS_INPUT));
```

Comment:
@subkrish This is your issue. Original code was directory, not file.
@romani This is another bad sign when tests pass but it is code coverage that tells us there is a problem.